examples/FakeIt/fakeit_test.cpp

```diff
@@ -0,0 +1,30 @@
+#include <fakeit/standalone/fakeit.hpp>
+#include <iostream>
+
+struct SomeInterface {
+	virtual int foo(int) = 0;
+	virtual int bar(std::string) = 0;
+};
+
+int main()
+{
+	using namespace fakeit;
+	
+	// Instantiate a mock object.
+	Mock<SomeInterface> mock;
+
+	// Setup mock behavior.
+	When(Method(mock,foo)).Return(1); // Method mock.foo will return 1 once.
+
+	// Fetch the mock instance.
+	SomeInterface &i = mock.get();
+
+	// Production code
+	i.foo(1);
+
+	// Verify method mock.foo was invoked.
+	Verify(Method(mock,foo));
+
+	// Verify method mock.foo was invoked with specific arguments.
+	Verify(Method(mock,foo).Using(1));
+}
```

Comment:
Please add new line here.

* http://0.readthedocs.io/en/latest/git.html#check-there-is-a-new-line-at-end-of-file